texturepacker/editor/src/texturepacker.clj

```diff
@@ -1261,41 +1263,65 @@
                            (g/transact
                              (concat
                                (g/operation-sequence op-seq)
-                               (g/operation-label "Add Animation")
+                               (g/operation-label (localization/message "operation.atlas.add-animation"))
```

Comment:
Not sure if we have a mechanism in place to localize strings in extensions yet? I.e. How it will work for community contributions, etc.

For now I use the same localized strings as we do for the Atlas resource type when there's an exact match, but haven't updated other strings yet.